atm.js

```diff
@@ -1,21 +1,74 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$( document ).ready(function() {
+    console.log( "ready!" );
+    var totalBalance = parseInt(checking + savings)
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-}
+  //add value of inputChecking to #checking
+  $("#checkingDeposit").click(function(){
+    var inputChecking = parseInt($("#inputChecking").val());
+    var checking = parseInt($("#checking").text().replace("$", ""));
+    var newBalance = checking + inputChecking
+    console.log(newBalance);
+    $("#checking").text("$" + newBalance)
+    if (checking > 0) {
+      $(".checkingAccount").removeClass("zero")
 
-function checkingWithdrawal(amount){
+    }
+  })
 
-}
+  //subtract inputChecking from #checking
+  $("#checkingWithdrawal").click(function(){
+    var inputChecking = parseInt($("#inputChecking").val());
+    var checking = parseInt($("#checking").text().replace("$", ""));
+    var savings = parseInt($("#savings").text().replace("$", ""));
+    var newBalance = checking - inputChecking
 
-function savingsDeposit(amount){
+    if((checking - inputChecking) < 0){
+      var overDraft = inputChecking - checking
+      $("#checking").text("$0")
+      var newSavingsBalance = (savings - overDraft)
+      $("#savings").text("$" + newSavingsBalance)
+      $(".checkingAccount").addClass("zero")
+    }
 
-}
+    //I could not get the savings to prevent a negative balance when overdraft
+    else if ((newBalance <= 0) || (newSavingsBalance <= 0)){
```

Comment:
2 things:

1. This shouldn't be in an else condition, because that means it will only run if the associated if (line 26 above) is false (i.e. if there was no need to overdraft). Another way to say this is that this bit of code (L35 - 37) should be nested inside the previous if statement.
2. You should only need to check if newSavingsBalance is less than 0 in that case.